src/wmnds/components/phase-indicator/_phase-indicator-link.njk

```diff
@@ -0,0 +1,17 @@
+{% macro wmndsPhaseIndicatorLink(params) %}
+<div class="wmnds-grid wmnds-banner-container">
+  <div class="wmnds-col-auto wmnds-float-left wmnds-m-r-xsm">
+     <a href="/" target="_self" class="wmnds-phase-indicator" title="Beta homepage of West Midlands Network Design System">
+       Beta
+     </a>
+  </div>
+  <div class="wmnds-col-auto">
+     <p class="wmnds-banner-container__text">
+        <a href="#" title="link title" target="_blank" class="wmnds-link">
+          Try our new disruptions page
+        </a>
```

Comment:
As tedious as it is, could import the wmndsLink njk file. So if the link gets updated in future, it will change here too.

https://github.com/wmcadigital/wmn-design-system/blob/master/src/wmnds/components/link/link/_link.njk

An example of the link being imported is here:
https://github.com/wmcadigital/wmn-design-system/blob/master/src/wmnds/patterns/banner/_banner.njk

Reply:
implemented